dts/bindings/timer/nxp,imx-gpt.yaml

```diff
@@ -16,3 +16,8 @@ properties:
 
     label:
       required: true
+
+    nxp,gptfreq:
+      type: int
+      required: true
+      description: gpt frequences
```

Comment:
@galak it's not clear to me when properties should have a prefix like this

Reply:
We haven't normally used such prefix in zephyr.  There's some convention of doing this in linux.